packages/corelib/src/dataModel/PartInstance.ts

```diff
@@ -25,9 +25,6 @@ export interface DBPartInstance extends InternalIBlueprintPartInstance {
 	/** Rank of the take that this PartInstance belongs to */
 	takeCount: number
 
-	/** Whether this instance was created because of RundownPlaylist.nextSegmentId. This will cause it to clear that property as part of the take operation */
-	consumesNextSegmentId?: boolean
```

Comment:
Personally, I think this should be mentioned in the migrations, even though the PartInstances are "temporary" (for consistency).

Reply:
This is not a trivial migration because of it being moved across collections. As it is such a minor property, and running migrations while a rundown is activated isnt the best idea anyway, I dont think this is worth spending time on